build-image/openhabian.conf

```diff
@@ -47,12 +47,12 @@ debugmode=on
 
 # OS Release
 # ubuntu, raspbian, debian
```

Comment:
would we need a raspios option and is_raspios() in helpers, too ?

Reply:
no, at least as of right now, I tested and the `etc/os-release` file reports as Rasbian still if it changes then we will but as of right now it is still the same.

Reply:
ok but please add them nonetheless

Reply:
I dont think we should because as has been outlined on the RPi fourms `raspios` is not the offical abbreviation and as that has not been decided yet see:

<https://www.raspberrypi.org/forums/viewtopic.php?f=66&t=275380&sid=1a468f226394ccddf4654a3d3d90cb7d#p1668248>

Reply:
alright, let's wait for them. Probably a trademark thingy once more ... But I wouldn't mind if you added is_raspios(), for now we do not use it